### DISCUSSION TITLE
Changing Live mode to different modes.

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#234] [Dev and Ideas]
Like I explained in issue #229, in 1.6.0 I reduced the number of call to the API as much as possible, and that just caused some of you to not be able to use the scoreboard the way you use to. and I'm going to work on adding different modes instead of just Live mode. 

I'm trying to determent what we call the "Use cases" here. How do you use the scoreboard in general. I will work on a couple modes depending on that. you can also throw some related ideas too.

**Comment by riffnshred** [comment]
For myself, I use it while I watch my fav teams play and while I do other things like work, play games, cook etc.... On off days I like to know who plays who and what are the standings and that's pretty much it. I don't count the moments I work on it here obviously.

**Comment by xTrackstar04x** [comment]
I am someone who has the screen on every night. Allows me to follow along with something that isn't my wife's stupid shows. Also I like to bet on Hockey, so I like to follow every game.  (not just my fav)

I think it would be cool to have a mixture of Live Games (Where its just stuck on your favorite team and updates everything on that game) and just the standard scrolling like it does when you team is not playing. So with this mode, it would still show you when teams are on power plays, When a goal is scored(?) but it would still cycle through the teams every 15 seconds.

**Comment by Coop21** [comment]
I have a Man Cave down in my basement with 3 tvs and I'll turn on the scoreboard when I'm down there watching sports.  For me I'd like the scores to refresh whether or not my team is playing.  I could get involved in some other project on my laptop and use the scoreboard to glance up and see what's going on in the league instead of having to pull out my phone and scroll through scores.  Also, I work from home and have an unlimited wifi plan, but I could see why those with data caps could be in for a surprise while using this board.

Thanks again for all your work on this.  It's been a lot of fun for me to tinker with it and really makes a cool addition to the Man Cave.  I'm already on planning on doing the MLB one as well.  

Just thought of something.  Would it be possible to add a scoring summary between periods for games involving favorited teams?  I don't know if that info is available to pull from the API.  But it might be nice to see who got the assist on goals since most of the time the assists aren't available right away.  Just an idea maybe for a future version.

**Comment by ddman2** [comment]
I have the scoreboard on all the time and made several others for individuals which I manage remotely.

When a preferred team is playing (some of us have multiple teams setup), I like to follow the top priority (higher preference) team’s game.  It comes in handy when I’m not in control of the tv selection or if I’m doing something else. I like to glance over and see score. Also the goal animations and penalty animations are an awesome touch. It makes for a great conversation piece. 

During intermission I like to get score updates from around the league. 

On off days, I like to know what games are on and what the scores are, but to be fair, I don’t really pay all that much attention to the other games.  I also have the team summary and standings show, but again, it’s mostly background.

**Comment by sflems** [comment]
Like @ddman2, I've got my board running all the time, and generally like having the live game state to takeover if one of my preferred teams are playing.  I limit the intermission state to just the NHL boards, but once that game is back on, its awesome when it's locked back in. Otherwise any other boards basically just loop outside of gametime.

Live games from preferred teams are the # 1 priority here! Top team is priority and once the preferred game ends, if another preferred team is found, it should display if live. 

Would be nice to have an option to cycle thru the live boards only vs only priority team vs all scores, but this is less important as the [GUI](https://github.com/sflems/nhl-led-scoreboard-webgui) makes switching teams on the fly a breeze 😉

**Reply by xTrackstar04x** [reply]
Love the little promo in there 😂

**Reply by sflems** [reply]
"You're gunna love my nuts!"
No Vince, Billy from cell 53 will.

**Comment by pkntl** [comment]
My board is on all the time in my home office. The live game board is the main use for me. During intermissions I'll have other scores displayed. In the post-game I'll go back to cycling between the other games, the division standings and the team summary. I use the same configuration for off-days as well.

The live game board is the most important to me. Otherwise the standings, other league games, and team summary are the ones I use when my preferred team isn't playing.

**Comment by bazookajoey** [comment]
I have the board setup in the home office and let it cycle through all of the standings, games for the day and weather. I have one preferred team and only get to see live mode on the rare afternoon game or move it near the tv.

During intermission, my board is setup to be a ticker for the other games being played, that’s it. 

I honestly wouldn’t change anything about how it works now. When I do get to see the board during game time, it’s perfect!

**Comment by infectedroot** [comment]
My board is setup in my living room and is used multiple times per day. Most of the time the board stays on the clock (5 minute time out) but also switches to weather and a bunch of custom boards for other things like stonks, pihole stats, etc. 

I only follow two teams. 

When games are on, the live board does it's thing. I also have the goal processing code sending a message to automation for audio and lights around the house.

During intermission it's just stats for the division and team. 

After the game, the score is good to review along with the regular set of boards in rotation. 

Hope this helps!

**Reply by WilliamSasson** [reply]
Are there any tutorials to make custom boards, such as stock boards? (This is my first project coding outside of a classroom using python/jupyter and would be cool to be able to make custom boards once I get the NHL scoreboard functioning)

**Comment by xxxDrewedxxx** [comment]
My 128 board is running all the teams as favorites, mostly because I like to see who is playing who.  Live game isn’t as important, mostly because some of the screens are screwed for 128 boards and I haven’t fixed them yet. 
I run a 64 board, but the is strictly on weather duty.  The 128 board take over that job come off season.   Perhaps I should just build another 128 board........

**Comment by kempdawg20** [comment]
My board is set up in my man cave (I'm a bachelor, my whole loft is a man cave), I pretty much keep it on in case a game gets interesting I'll know to jump on. Plus it's just fun to have updating as games go on.
